# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openoffice-org/portable.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- OOO320_m12/solenv/bin/make_installer.pl.orig	2010-03-12 10:52:17.982909051 +0000
+++ OOO320_m12/solenv/bin/make_installer.pl	2010-03-12 10:55:19.806910741 +0000
@@ -1371,7 +1371,7 @@
 			# try it again later.
 			####################################################
 
-			if (( $installer::globals::patch ) || ( $installer::globals::languagepack ) || ( $installer::globals::packageformat eq "native" ) || ( $installer::globals::packageformat eq "osx" )) { $allvariableshashref->{'POOLPRODUCT'} = 0; }
+			if (( $installer::globals::patch ) || ( $installer::globals::languagepack ) || ( $installer::globals::packageformat eq "native" ) || ( $installer::globals::packageformat eq "portable" ) || ( $installer::globals::packageformat eq "osx" )) { $allvariableshashref->{'POOLPRODUCT'} = 0; }
 
 			if ( $allvariableshashref->{'POOLPRODUCT'} )
 			{
--- OOO320_m12/solenv/bin/modules/installer/control.pm.orig	2010-03-12 15:36:08.268404990 +0000
+++ OOO320_m12/solenv/bin/modules/installer/control.pm	2010-03-12 15:37:06.718906243 +0000
@@ -715,6 +715,7 @@
 	if ( $installer::globals::patch ) {	$installer::globals::addsystemintegration = 0; }
 	if ( $installer::globals::languagepack ) { $installer::globals::addsystemintegration = 0; }
 	if ( $installer::globals::packageformat eq "native" ) { $installer::globals::addsystemintegration = 0; }
+	if ( $installer::globals::packageformat eq "portable" ) { $installer::globals::addsystemintegration = 0; }
 
 	my $infoline = "Value of \$installer::globals::addsystemintegration: $installer::globals::addsystemintegration\n";
 	push( @installer::globals::globallogfileinfo, $infoline);
